src/storagemon/storagemon.cpp

```diff
@@ -0,0 +1,160 @@
+/*
+ * This file is part of crash-reporter
+ *
+ * Copyright (C) 2021 Jolla Ltd.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA
+ *
+ */
+
+#include <QCommandLineParser>
+#include <QCoreApplication>
+#include <QDebug>
+#include <QTextStream>
+#include <QTimer>
+#include <QTranslator>
+
+#include <notification.h>
+
+#include "creporterapplicationsettings.h"
+#include "creportercoreregistry.h"
+#include "creporternamespace.h"
+#include "creportersavedstate.h"
+#include "creporterutils.h"
+
+#include "creporterlogger.h"
+
+const int DEFAULT_SIZE_LIMIT_MB = 100;
+
+QTextStream &qerr()
+{
+    static QTextStream qerr(stderr);
+    return qerr;
+}
```

Comment:
This looks strange. Just do qWarning()?

Reply:
Using right tool for the right job :)

Reply:
How's this right? qWarning() is defined to output to stderr if no error handler is set, and this is used only for a single error string. Just using that should effectively do the same thing.

Reply:
This is a CLI tool. The (primary) UI is CLI. QDebug is good for logging -- debug output, messages about errors coming from the environment (something not directly caused by the user), etc. -- but not for the UI. Not for the regular dialog with the user, like asking them for input, telling them about using the program a wrong way or informing them of the overall execution outcome.

Reply:
I'm left to guess what you mean since you're kind of saying the internally used api should matter for the user instead of what the user sees. But reminds me the QT_MESSAGE_PATTERN might affect the output if that's what you're referring to.
Then again it's libexec so it's not even meant to be run directly.

But I don't really care that much to get stuck on this more.

Reply:
Yes, it is message pattern, it is filtering with logging rules which is not desired, and it is also the quoting of QString imposed by default which needs explicit `noquote()`, similarly implicit spacing etc.